pkg/api/v1beta3/types.go

```diff
@@ -177,7 +177,7 @@ type Volume struct {
 	// Source represents the location and type of a volume to mount.
 	// This is optional for now. If not specified, the Volume is implied to be an EmptyDir.
 	// This implied behavior is deprecated and will be removed in a future version.
-	VolumeSource `json:",inline,omitempty"`
+	VolumeSource `json:"inline,omitempty" description:"location and type of volume to mount; at most one of HostDir, EmptyDir, GCEPersistentDisk, or GitRepo; default is EmptyDir"`
```

Comment:
There does need to be a leading comma, actually. Also, an inline field shouldn't have a description, since it won't map to a single field in the json. Please revert this line.